use-timescale/continuous-aggregates/create-a-continuous-aggregate.md

```diff
@@ -26,9 +26,31 @@ example, if the original hypertable's chunk time interval is 7 days, the
 continuous aggregates that are on top of it have a 70 day chunk time
 interval.
 
+## Function support
+
+In TimescaleDB 2.7 and later, continuous aggregates support all PostgreSQL
+aggregate functions. This includes both parallelizable aggregates, such as `SUM`
+and `AVG`, and non-parallelizable aggregates, such as `RANK`.
+
+In TimescaleDB&nbsp;2.10.0 and later, the `FROM` clause supports `JOINS`, with
```

Comment:
I think it's not the FROM clause that supports the JOIN, but the "CREATE MATERIALIZED VIEW" clause for continuous aggregates.